pkg/trace/trace.go

```diff
@@ -255,6 +255,25 @@ func AddInfo(ctx context.Context, args ...log.Marshaler) {
 	}
 }
 
+// Error is a convenience for attaching an error to a span.
+func Error(ctx context.Context, err error) error {
```

Comment:
Why not make this simply `AddInfo(ctx, events.NewErrorInfo(err))`?

Reply:
Likely a distinct concern, but because I'm not sure what to make of traces with fields like: https://ui.honeycomb.io/outreach-a0/datasets/outreach/result/3fhi6d8M9A6?tab=traces

My thought was to provide something dead simple. Maybe it's worse though? Seems like it might be better to simply stick to the existing types and lead the loggableError type out.

Reply:
Yeah, it is a bit much, but I'm not sure there's a great alternative.  Sometimes the key to understanding what's wrong actually is buried 5 causes deep.

Reply:
I think regardless, it's an unrelated issue, so I'm just going to use the existing ErrorInfo type and try to formulate some more precise thoughts on how the error gets serialized.

The last thing I'll say about it here is I think this format makes some important questions more difficult to answer, like what is the error distribution for my service?

Reply:
Fixed in [2187936](https://github.com/getoutreach/gobox/pull/258/commits/21879361059969b58cecc80dd9d010320e201bb5)